docs/source/howto/export_environment.rst

```diff
@@ -26,6 +26,24 @@ be resolved (``ResolvePackageNotFound``).
 The solution
 ============
 
+Export your explicit install commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you only want to have an ``environment.yml`` that reflects the
```

Comment:
How about something like the following? I think we should help unfamiliar users by telling them that they want to do this over other things (unless they know what they are doing, in which case they are probably not reading this).

To get a minimal ``environment.yml`` that only contains the packages you explicitly installed run ``conda env export --from-history environment.yml``. We recommend that you use this option to create your ``environment.yml``. The resulting ``environment.yml`` then contains ...

Reply:
Thanks, integrated this.